test/unit/service/invoice-service.ts

```diff
@@ -80,7 +80,7 @@ export async function requestToTransaction(
   let total = 0;
   await Promise.all(
     transactionRequests.map(async (t) => {
-      const transactionResponse = await TransactionService.createTransaction(t);
+      const transactionResponse = await (new TransactionService()).createTransaction(t);
```

Comment:
You don't need the brackets around `new TransactionService()`. You do this in a lot of places, so I'll skip marking them everywhere.